app/views/organisations/_header.html.erb

```diff
@@ -14,6 +14,11 @@
           <%= t('organisations.part_of') %>
           <%= @show.parent_organisations %>
         </div>
+      <% elsif @organisation.is_court_or_hmcts_tribunal? %>
+        <div class="organisation__parent-organisations brand--<%= @organisation.brand %>">
```

Comment:
Do courts ever have a brand? If not we can remove `brand--<%= @organisation.brand %>` here.